Komodo/Assets/Packages/KomodoCore/Runtime/Scripts/RuntimeSession/TabButton.cs

```diff
@@ -50,6 +50,23 @@ void Start ()
         public void Select ()
         {
             onTabSelected.Invoke();
+
+            if (gameObject.name == "Settings") 
```

Comment:
TODO: move each of these code blocks into the onTabSelected Action for each tab button, and out of this general TabButton class

Reply:
Like this:

Inside KomodoMenu.cs >  Start:
```


            settingsTab.onTabSelected.AddListener(() => 
            {
                UIManager.Instance.SendMenuVisibilityUpdate(...);
            })
 ```